### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by ariachy**
Minor bug appeared when ticketswap login modal shows the web version of the plugin. The id of the login button is different. The if elif statement should solve this issue.

### CODE REVIEW

**Comment by davisnando** [comment]
Still check your pull request because there is still some double code

File: main.py [hunk 1]
```diff
@@ -43,6 +43,18 @@ def login(self):
...
+
+                time.sleep(1)
+        
+        elif driver.find_element_by_id('u_0_0'):
+                send_login = driver.find_element_by_id('u_0_0')
+                send_login.click()
+
+                time.sleep(1)
 
         send_login = driver.find_element_by_id('u_0_0')
```
[the remark this answers is not in the document]
Comment (ariachy) [reply main.py]: No idea what's wrong to be honest. But I removed all the lines under if and kept the send_login. Sorry but I am new to both Github and Python...

[the remark this answers is not in the document]
Comment (davisnando) [reply main.py]: line 54 and 55 is the same as line 59 and line 60

**Comment by ariachy** [comment]
There is another issue with the code. Whenever a ticket appears on ticketswap, it gives this error: IndexError: list index out of range on item.findAll('a')[0]

**Pull request merged**